go.mod

```diff
@@ -4,12 +4,14 @@ go 1.13
 
 require (
 	github.com/asaskevich/govalidator v0.0.0-20190424111038-f61b66f89f4a // indirect
+	github.com/craicoverflow/go-environment-variables-example v0.0.0-20190228224948-ee3fabeff275
```

Comment:
What is the reason behind adding this dependency here?

Reply:
fix applied.